tsconfig.base.json

```diff
@@ -15,6 +15,7 @@
     "resolveJsonModule": true,
     "composite": true,
     "baseUrl": ".",
+    "skipLibCheck": true,
```

Comment:
I would like to know more about the types of errors you were seeing

Reply:
Similar to this: https://stackoverflow.com/questions/65353067/error-ts6200-error-ts2403-definitions-of-the-following-identifiers-conflict-w but between anchor and web3.js

Reply:
I tried bumping versions but that didn't help 🤷‍♀️

Reply:
I mean the actual errors. Are they just type definition errors? How many of them?

Reply:
On running `yarn build`
![Screenshot_20221230_115045](https://user-images.githubusercontent.com/91517002/210024322-fc5d15bb-2520-48a2-ac4d-2e1d0ac4e843.png)
![Screenshot_20221230_115105](https://user-images.githubusercontent.com/91517002/210024326-06872dc7-21a7-44b4-bd14-5f859b66da82.png)

Reply:
Ah yes. I'm familiar with that specific wall of errors. Thanks.